languages/java/code/.gitignore

```
@@ -0,0 +1,3 @@
+java_{{course_slug}}.jar
```

```suggestion
*.jar
```

(Let's be safe and not have any large files committed)